app/src/interfaces/CCApp.ts

```diff
@@ -2,7 +2,8 @@ export declare module CCApp {
 
   export interface Datum {
     approved: boolean;
-    id: string;
+    rejected: boolean;
```

Comment:
Right now we could technically end up with an object being both approved and rejected

Reply:
We have also discussed this on the backend PR. In short, having two separate fields allows for more flexibility in determining what and how. But for sure, having just one field can make certain aspects easier, we agree.
But given where we are at in the development process, I feel that it might not take priority for us right now.